plugins.audio.pbsfm/resources/language/resource.language.en_gb/strings.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<strings>
+	<!-- Plugin name -->
+	<string id="30000">PBS FM Live stream</string>
+	<string id="30001">Ondemand</string>
+</strings>
```

Comment:
@anemes those lines shouldn't be in this file and the file has to be named `strings.po`.
Please submit a updated version of your addon.